packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -239,8 +244,25 @@ provideUseId(() => {
   return `${ATTR_KEY}-${instanceId}`
 })
 
-// Provide global security
-provide(GLOBAL_SECURITY_SYMBOL, () => props.parsedSpec.security)
+// Create the workspace store and provide it
+console.time('populate workspace')
+const workspaceStore = createWorkspaceStore({
+  isReadOnly: true,
+  proxyUrl: props.configuration.proxy,
+  themeId: props.configuration.theme,
+  useLocalStorage: false,
+})
+// Populate the workspace store
+workspaceStore.importSpecFile(props.parsedSpec, 'default', {
```

Comment:
Moved it back to draft due to this, @hanspagel maybe we callaborate but need a way to pass the spec into the importer